cedar-policy-core/src/extensions.rs

```diff
@@ -124,105 +214,75 @@ impl<'a> Extensions<'a> {
     /// `Err` is returned in the case that multiple constructors have that signature.
```

Comment:
Comment needs updating here too. (and the polymorphic comment no longer applies)